app/src/main/java/de/danoeh/antennapod/activity/OnlineFeedViewActivity.java

```diff
@@ -154,12 +154,18 @@ protected void onCreate(Bundle savedInstanceState) {
             throw new IllegalArgumentException("Activity must be started with feedurl argument!");
         }
 
-        Log.d(TAG, "Activity was started with url " + feedUrl);
-        setLoadingLayout();
-        if (savedInstanceState == null) {
-            startFeedDownload(feedUrl, null, null);
-        } else {
-            startFeedDownload(feedUrl, savedInstanceState.getString("username"), savedInstanceState.getString("password"));
+        if(feedUrl == null){
```

Comment:
Please add a space before the opening bracket

Reply:
You removed a space instead of adding one. Now there are two spaces missing ;)

Reply:
sorry about that. thanks